slides.md

```diff
@@ -332,3 +332,36 @@ To alter the state of a component, you need to invoke `this.setState` inside tha
 The second case is required when your update depends on the current state. The function receives the current state and must return the new state (which is then merged, like the previous case). Since state updates are asynchronous, this is necessary because you're not ensured to have the latest state inside `this.state`.
 
 Add a callback to the various components, and update the state inside `App`.
+
+# Promises
+
+The standard interface to do asynchronous work in JS is `Promise`. A promise is an object representing something that will end in the future.
+
+## Receiving results
+
+To access the results of a promise (the promise was **resolved**), you need invoke the `then` method and provide a callback to it. The callback will receive the result of the operation as the only parameter.
```

Comment:
Need *to* invoke. 
Also the second sentence might not immediately be clear to someone who isn't familiar with async operations.

Reply:
👍 for the grammar error. the second part, I'm assuming knowledge as a software engineer, just not knowledge in JS

Reply:
4d5ef38